select-xamarin-sdk-v2.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -e -o pipefail
+
+echo "Set SDK"
+
+get_framework_path() {
+  local framework=$1
+  case $framework in
+  mono) echo '/Library/Frameworks/Mono.framework/Versions' ;;
+  ios) echo '/Library/Frameworks/Xamarin.iOS.framework/Versions' ;;
+  android) echo '/Library/Frameworks/Xamarin.Android.framework/Versions' ;;
+  mac) echo '/Library/Frameworks/Xamarin.Mac.framework/Versions' ;;
+  *) ;;
+  esac
+}
+
+set_current_folder() {
+  local framework=$1
+  local version=$2
+
+  echo "framework = $framework, version = $version"
+
+  if [ ! -z ${framework} ]; then
+    local countDigit=$(echo "${version}" | grep -o "\." | grep -c "\.")
+    if [[ ! countDigit -eq 1 ]]; then
+      echo "Wrong framework's versions."
+      return
+    fi
+  fi
+
+  local FOLDER=$(get_framework_path "$framework")
```

Comment:
May be `framework_path` will be better name for this variable.
Also, could you please use consistent approach for variables?
`folder` or `FOLDER` never mind but it should be consistent

Reply:
`local framework_path=$(get_framework_path "$framework")`